unfetter-discover/unfetter
# Issue 1401

When creating the private-config and local-settings.json, the uac.yml variable for auth_services was causing the ansible templating system to improperly build the template.

Was: 
auth_services:[
"[",
""",
"g",
"i",
etc

TO TEST
Remove the files in "backup/private-vol"
Stop all the container
remove the containers
remove the volume private-vol
ansible-playbook deploy-dev.yml

The private-config.json file should look correct.
